apps/src/templates/studioHomepages/BorderedCallToAction.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '../../util/color';
```

Comment:
nit: use absolute paths instead of relative paths for imports unless you're importing files in your same directory